.github/CODEOWNERS

```diff
@@ -2,6 +2,7 @@
 
 /internal/resources/grafana/*_alerting_* @grafana/terraform-provider @grafana/alerting-squad
 /internal/resources/cloud/* @grafana/terraform-provider @grafana/grafana-com-maintainers
+/internal/resources/cloudobservability/* @grafana/terraform-provider @grafana/middleware-apps
```

Comment:
For the @grafana/terraform-provider team, I think that someone from your team needs to give [this team](https://github.com/orgs/grafana/teams/middleware-apps/repositories) write access to this repo, in order to resolve this error. Please let me know if something different needs to be done.